RentACar/RentACar/Models/Rental.cs

```diff
@@ -0,0 +1,19 @@
+﻿using RentACar.Models.Base;
+using System;
+
+namespace RentACar.Models
+{
+    public class Rental : Entity
+    {
+        public TimeSpan RentalDuration => MyMethodTime();
+        public Client Client { get; set; }
+        public Car Car { get; set; }
+        public DateTime RentalDate { get; set; }
+        public DateTime ReturnDate { get; set; }
+
+        private TimeSpan MyMethodTime()
```

Comment:
we may want to find a better name for this method, don't you agree?

Reply:
Oh, horrible name! I forgot to change it.. thank you for noticing that!